File: fintechplatform-ui/src/main/java/com/fintechplatform/ui/iban/ui/IBANActivityModule.java
package com.fintechplatform.ui.iban.ui;

import dagger.Module;
import dagger.android.ContributesAndroidInjector;
@Module
public abstract class IBANActivityModule {
    @ContributesAndroidInjector
    abstract IBANActivity contributeIBANActivity();
}

File: fintechplatform-ui/src/main/java/com/fintechplatform/ui/iban/ui/IBANActivityModule.kt
package com.fintechplatform.ui.iban.ui

import dagger.Module
import dagger.android.ContributesAndroidInjector

@Module
abstract class IBANActivityModule {
    @ContributesAndroidInjector(modules = [IBANFragmentModule::class])
    abstract fun contributeIBANActivity(): IBANActivity
}